.circleci/config.yml

```diff
@@ -0,0 +1,45 @@
+version: 2
+jobs:
+  build:
+    parallelism: 1
+    working_directory: ~/ezcater
+    docker:
+      - image: circleci/ruby:2.4
+    steps:
+      - checkout
+
+      # Restore bundle cache
+      - restore_cache:
+          keys:
+            - ruby-cache-{{ .Branch }}-{{ checksum "Gemfile" }}
+            - ruby-cache-{{ .Branch }}-
+            - ruby-cache-
+
+      # Bundle install dependencies
+      - run: bundle check --path=vendor/bundle || bundle install --path=vendor/bundle --jobs=4 --retry=3
+      - run: bundle clean --force
+
+      # Store bundle cache
+      - save_cache:
+          key: ruby-cache-{{ .Branch }}-{{ checksum "Gemfile" }}
+          paths:
+            - vendor/bundle
+
+      # Run Rubocop
+      - run:
+          name: RuboCop
+          command: ruby bin/circle_rubocop.rb
+
+      # Run rspec in parallel
+      - type: shell
+        command: |
+          bundle exec rspec --profile 10 \
+                            --out test_results/rspec.xml \
+                            --format progress \
+                            $(circleci tests glob "spec/**/*_spec.rb" | circleci tests split --split-by=timings)
```

Comment:
Similarly here, we probably don't need to worry about splitting, and probably will never reach the point of needing to parallelize these specs.